combiner/lists/AM_SVG-full.lis

```diff
@@ -0,0 +1,14 @@
+jax/input/AsciiMath/config.js
+jax/output/SVG/config.js
+jax/output/FastHTML/config.js
+extensions/asciimath2jax.js
+extensions/MathEvents.js
+extensions/MathZoom.js
+extensions/MathMenu.js
+jax/element/mml/jax.js
+extensions/toMathML.js
```

Comment:
The `-full` version should include `jax/input/AsciiMath/jax.js` in addition to the `config.js` file.

In fact, even the versions without `-full` have included the input jax in the past.  They probably shouldn't have, but they all do, so I don't know if you want to do that for `AM_SVG` below or not (it is included in `AM_CHTML` above, so I would think so).